### DISCUSSION TITLE
What does wpf's DependencyProperty's PropertyChangedCallback correspond to in avalonia?

### DISCUSSION THREAD

**Discussion by nir34** [AvaloniaUI/Avalonia#13962] [Q&A] [answered]
StyledProperty doesn't seem to have a ChangedCallback parameter.

**Comment by timunie** [comment, marked as answer]
Listen to PropertyChanged event of any control. In eventargs you get the property info

**Reply by timunie** [reply]
To add if you create a custom control, simply override OnPropertyChanged
